### PR TITLE
implement correct name for based auction

### LINKED ISSUE #441: Display correct item name for reference auctions
Currently the title of the new item is used for all of them which is wrong

### PR DESCRIPTION
**Pull request by matthias-luger**
fixes #441

### CODE REVIEW

**Comment by matthias-luger** [comment]
It would be cool if the property could be renamed from ItemName to itemName like it is everywhere else.

**Comment by Ekwav** [comment]
It is not called `ItemName`, it is called `name` should I rename it to `itemName` ?

**Comment by Ekwav** [comment]
also this MR targets should target `dev` because the property is only available in commands

**Comment by Ekwav** [comment]
also this MR targets should target `dev` because the property is only available in commands

File: src/utils/Parser/APIResponseParser.tsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ export function parseAuction(auction: any): Auction {
         end: new Date(auction.end),
         item: {
             tag: auction.tag,
-            name: auction.itemName
+            name: auction.itemName || auction.ItemName
```
Comment (Ekwav) [inline src/utils/Parser/APIResponseParser.tsx:34]: > It is not called `ItemName`, it is called `name` should I rename it to `itemName` ?
https://github.com/Coflnet/hypixel-react/pull/456#issuecomment-917605441

**Review by Ekwav — CHANGES_REQUESTED** [review]

**Review by Ekwav — APPROVED** [review]

**Pull request merged**